apidocs/overview.md

```diff
@@ -0,0 +1,438 @@
+    <p><a href="">Update expiring user targets on feature flag</a></p>
+    <p><a href="">Update expiring user target for flags</a></p>
+</blockquote>
+
+# Errors
+
+The API always returns errors in a common format. Here's an example:
+
+```json
+{
+    "code": "invalid_request",
+    "message": "A feature with that key already exists",
+    "id": "30ce6058-87da-11e4-b116-123b93f75cba"
+}
+```
+
+The general class of error is indicated by the `code`. The `message` is a human-readable explanation of what went wrong. The `id` is a unique identifier-- use it when you're working with LaunchDarkly support to debug a problem with a specific API call.
+
+## HTTP Status - Error Response Codes
+
+| Code | Definition | Desc. | Possible Solution |
+| ---- | ---------- | ----- | ----------------- |
+| 400  | Bad Request | A request that fails may return this HTTP response code | Ensure JSON syntax in request body is correct. |
+| 401  | Unauthorized | User doesn't have permission to an API call | Ensure your SDK key is good. |
+| 403  | Forbidden | User does not have permission for operation. | Ensure that the user or access token has proper permissions set. |
+| 409  | Conflict | The API request could not be completed because it conflicted with a concurrent API request. | Retry your request. |
+| 429  | Too many requests | See [Rate limiting](ref:rate-limiting) | Wait and try again later. |
+
+# CORS
+
+The LaunchDarkly API supports Cross Origin Resource Sharing (CORS) for AJAX requests from any origin.  If an `Origin` header is given in a request, it will be echoed as an explicitly allowed origin. Otherwise, a wildcard will be returned: `Access-Control-Allow-Origin: *`.  For more information on CORS, see the [CORS W3C Recommendation](http://www.w3.org/TR/cors).  Example CORS headers might look like:
+
+```http
+Access-Control-Allow-Headers: Accept, Content-Type, Content-Length, Accept-Encoding, Authorization
+Access-Control-Allow-Methods: OPTIONS, GET, DELETE, PATCH
+Access-Control-Allow-Origin: *
+Access-Control-Max-Age: 300
+```
+
+You can make authenticated CORS calls just as you would make same-origin calls, using either [token or session-based authentication](doc:authentication). If you’re using session auth, you should set the `withCredentials` property for your `xhr` request to `true`. Remember-- you should never expose your access tokens to untrusted users.
+
+# Rate limiting
+
+We use several rate limiting strategies to ensure the availability of our APIs. Rate-limited calls to our APIs will return a `429` status code. Calls to our APIs will include headers indicating the current rate limit status. The specific headers returned depend on the API route being called-- the limits differ based on the route, authentication mechanism, and other factors. Routes that are not rate limited may not contain any of the headers described below.
+
+<blockquote>
+    <h3><span>❗️</span>Rate limiting and SDKs</h3>
```

Comment:
Hmm this is an h3 -- I wonder how this looks with and impacts our headers and nav.

Reply:
Great question. I experimented to see. Only markdown headers are added to nav, and only to two levels (i.e. `#` and `##`). Explicit html header tags are not added to nav, nor are h3s anyway.